pages/1.11/security/ent/perms-management/index.md

```diff
@@ -36,3 +36,11 @@ To manage permissions for **users** from the DC/OS Enterprise CLI, use the follo
 
 * `dcos security org users grant [OPTIONS] UID RID ACTION`
 * `dcos security org users revoke [OPTIONS] UID RID ACTION`
+
+### Managing permissions using the API
+
+The [IAM HTTP API](/1.11/security/ent/iam-api/) provides operations to manage permissions for users and groups.
+
+Note that all entities must exist when creating a permission.
+
+A typical pattern is to create a permission is to call `PUT /acls/{rid}` first, ignoring any 409 failure status that indicates that it already exists, before calling `PUT /acls/{rid}/users/{uid}/{action}`  or `PUT /acls/{rid}/groups/{gid}/{action}`.
```

Comment:
> A typical pattern is to create a permission is

remove one "is"

Reply:
>  is to call `PUT /acls/{rid}` first 

maybe add: `(to create the access control list for the protected resource {rid})`

Reply:
Made the above modifications and put the two steps in point form to clarify further.